params_online.go

```diff
@@ -33,6 +33,20 @@ type OnlineQueryParamsComplete struct {
 	underlying OnlineQueryParams
 }
 
+// WithQueryName returns a copy of Online Query parameters with the specified input added.
+// For use via method chaining. See OnlineQueryParamsComplete for usage examples.
+func (p OnlineQueryParamsComplete) WithQueryName(queryName string) OnlineQueryParamsComplete {
+	p.underlying = p.underlying.WithQueryName(queryName)
+	return p
+}
+
+// WithQueryNameVersion returns a copy of Online Query parameters with the specified input added.
+// For use via method chaining. See OnlineQueryParamsComplete for usage examples.
+func (p OnlineQueryParamsComplete) WithQueryNameVersion(queryNameVersion string) OnlineQueryParamsComplete {
+	p.underlying = p.underlying.WithQueryNameVersion(queryNameVersion)
+	return p
+}
+
```

Comment:
I think for non `Feature`-requiring params we're not adding `withX` methods for it, and instead just requiring users to do `OnlineQueryParams{QueryNameVersion: "abc"}` because otherwise we'd have to add a `withX` method for every param, and for every `OnlineQueryParam/OnlineQueryParamWithInput/OnlineQueryParamWithOutput`